packages/cli/src/commands/inspect.ts

```diff
@@ -165,19 +184,24 @@ export default async function main(client: Client) {
   return 0;
 }
 
-// renders the state string
 function stateString(s: Deployment['readyState']) {
+  const CIRCLE = '● ';
+  const sTitle = s && title(s);
   switch (s) {
     case 'INITIALIZING':
-      return chalk.yellow(s);
-
+    case 'BUILDING':
+    case 'DEPLOYING':
+    case 'ANALYZING':
+      return chalk.yellow(CIRCLE) + sTitle;
     case 'ERROR':
-      return chalk.red(s);
-
+      return chalk.red(CIRCLE) + sTitle;
     case 'READY':
-      return s;
-
+      return chalk.green(CIRCLE) + sTitle;
+    case 'QUEUED':
+      return chalk.white(CIRCLE) + sTitle;
+    case 'CANCELED':
+      return chalk.gray(sTitle);
```

Comment:
```suggestion
      return chalk.gray(CIRCLE) + sTitle;
```

I think this should say canceled, right?